Description: avoid dependencies
Author: Xavier Guimard <yadd@debian.org>
Forwarded: not-needed
Last-Update: 2020-12-08

Index: node-object-inspect/test/indent-option.js
===================================================================
--- node-object-inspect.orig/test/indent-option.js
+++ node-object-inspect/test/indent-option.js
@@ -1,10 +1,9 @@
 var test = require('tape');
-var forEach = require('for-each');
 
 var inspect = require('../');
 
 test('bad indent options', function (t) {
-    forEach([
+    [
         undefined,
         true,
         false,
@@ -13,7 +12,7 @@ test('bad indent options', function (t)
         Infinity,
         -Infinity,
         NaN
-    ], function (indent) {
+    ].forEach(indent => {
         t['throws'](
             function () { inspect('', { indent: indent }); },
             TypeError,
Index: node-object-inspect/test/inspect.js
===================================================================
--- node-object-inspect.orig/test/inspect.js
+++ node-object-inspect/test/inspect.js
@@ -1,7 +1,6 @@
 var test = require('tape');
 var hasSymbols = require('has-symbols/shams')();
 var utilInspect = require('../util.inspect');
-var repeat = require('string.prototype.repeat');
 
 var inspect = require('..');
 
@@ -78,7 +77,7 @@ test('maxStringLength', function (t) {
         'maxStringLength must be >= 0, or Infinity, not negative'
     );
 
-    var str = repeat('a', 1e8);
+    var str = 'a'.repeat(1e8);
 
     t.equal(
         inspect([str], { maxStringLength: 10 }),
